test/fixtures/dumpfixture.sh

```diff
@@ -1 +0,0 @@
-pg_dump --host localhost --port 5432 --username "postgres" --no-password  --format plain --column-inserts --verbose --file "./test/fixtures/schema.sql" "postgrest_test"
```

Comment:
I don't think anyone does (or should for that matter!) use this command anymore. Am I wrong?